styles/pr-comment.less

```diff
@@ -58,4 +58,86 @@
     vertical-align: middle;
   }
 
+
+  // Suggested changes
+  .js-suggested-changes-blob {
```

Comment:
I assume we don't have much control over the markup that is coming from the GitHub API?

![screen shot 2019-02-27 at 3 28 47 pm](https://user-images.githubusercontent.com/378023/53470302-8b710c00-3aa4-11e9-8d4a-208da79c35bb.png)

`.js-suggested-changes-blob` shouldn't be used for styling, but it seems like the best identifier to scope "suggested changes". Also, the class names from .com could change anytime and might break the styling in Atom.

Reply:
yeah, I thought about how styling based on this classname is fragile 😦 

On the plus side, no functionality would be broken...it would just look janky again.

The "right" way to do this is to identify "suggested changes" comments based on data attributes that come from the graphQL api (which is probably possible, but I haven't looked at the schema.) . Then we apply our own classname.